lib/capistrano/tasks/fiesta.rake

```diff
@@ -24,7 +24,11 @@ namespace :fiesta do
   end
 
   def build_report
-    Capistrano::Fiesta::Report.new(repo_url, report_options)
+    Capistrano::Fiesta::Report.new(repo, report_options)
+  end
+
+  def repo
+    repo_url.slice(/github.com[:\/](\S+\/\S+)\.git/, 1)
```

Comment:
Since this has now moved to a `rake` file making it harder to test, what do you think about extracting it to a `RepoUrlParser` or something like that, so that we can write a test for it?

Reply:
Sounds good to me, d5e5a05 ♻️